enterprise/admin/__init__.py

```diff
@@ -382,11 +382,11 @@ def get_readonly_fields(self, request, obj=None):
             return readonly_fields + tuple(get_all_field_names(self.model))
         return readonly_fields
 
-    def get_enrolled_course_string(self, course_ids):
+    def get_enrolled_course_string(self, course_ids, enterprise_customer_user):
         """
         Get an HTML string representing the courses the user is enrolled in.
         """
-        courses_client = CourseApiClient()
+        courses_client = CourseApiClientJwt(enterprise_customer_user.user)
         course_details = []
         for course_id in course_ids:
             name = courses_client.get_course_details(course_id)['name']
```

Comment:
@sameenfatima78 `courses_client.get_course_details(course_id)['name']` will raise an exception if `get_course_details` function returns None. Is this intended behaviour?

Reply:
Yes, in that case it doesn't throw any exception, it just converts `None` into string and shows that against the course details. However, I have handled this case now. Instead of `None,` it will now show "No course details found for {course_id}".